packages/credentials/src/V1/KiltCredentialV1.ts

```diff
@@ -385,6 +448,35 @@ export async function validateSubject(
       [key.substring(vocab.length)]: value,
     }
   }, {})
-  // validates against CType (also validates CType schema itself)
-  CType.verifyClaimAgainstSchema(claims, cType)
+
+  try {
+    const references = extractUniqueReferences(cType)
+
+    const referencedCTypes = await Promise.all(
+      Array.from(references).map(async (ref) => {
+        try {
+          const referencedCType = await cachingCTypeLoader(ref as any)
+          return referencedCType
+        } catch (error) {
+          // eslint-disable-next-line no-console
+          console.error(`Failed to fetch CType for reference ${ref}:`, error)
```

Comment:
we don't use console statements in library code, that's why the lint rule exists

Reply:
Fixed at: https://github.com/KILTprotocol/sdk-js/pull/919/commits/bbfced24e1897748e57f40ddc5fa06267ed4962c